composer.json

```
@@ -27,7 +27,7 @@
     },
     "require": {
         "php": ">=5.4.2",
-        "guzzlehttp/psr7": "^1.0"
+        "guzzlehttp/psr7": "^2.0"
```

```suggestion
        "guzzlehttp/psr7": "^1.0 || ^2.0"
```

There is no reason this library needs to force 2.0 only, nothing about the classes that it's using has changed so significantly that both versions can't be supported for a while.

---

of course. my bad!

thank you.